packages/MarkdownEditor-Core.package/MarkdownBlockTextStyler.class/instance/privateStyle..st

```diff
@@ -0,0 +1,6 @@
+parsing
```

Comment:
Hmm, the category seams to be wrong. `SHTextStyler` puts `privateStyle:` into `private`, whereas I would put `privateStyle:` and `buildAttributeRunsFrom:` into the same category, for example `styling`. What category do you think is best?

Same problem in the `InlineTextStyler`

Reply:
`styling` would be fine with me.